docs/pages/versions/unversioned/sdk/auth-session.md

```diff
@@ -142,9 +147,20 @@ Cancels an active `AuthSession` if there is one. No return value, but if there i
 
 ### `AuthSession.getRedirectUrl()`
 
-Get the URL that your authentication provider needs to redirect to. For example: `https://auth.expo.io/@your-username/your-app-slug`.
+```ts
+AuthSession.getRedirectUrl(extraPath?: string): string
+```
+
+Get the URL that your authentication provider needs to redirect to. For example: `https://auth.expo.io/@your-username/your-app-slug`. You can pass an additional path component to be appended to the default redirect URL.
 
```

Comment:
I think it will be better to do it using formatting which is consistent with the rest of the document. It should look like this:
```
### `AuthSession.getRedirectUrl(extraPath)`

<description>

**Arguments**
-extraPath: __String__ - ...

**Returns**
- string ...
```
(Something similar to this https://docs.expo.io/versions/latest/sdk/auth-session/#authsessionstartasyncoptions)